spec/lib/record_redis_queue_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+
+module SupplejackApi
+  describe RecordRedisQueue do
+    let(:buffer) { SupplejackApi::RecordRedisQueue.new }
```

Comment:
Is this necessary? 
Isn't it the subject and is set up implicitly with `describe RecordRegisQueue`?

Reply:
Potentially! This story was just about moving the code, so I didn't actually alter any of it apart from some of the names.